contrib/bitcoinz-cli.bash-completion

```diff
@@ -1,10 +1,12 @@
-# bash programmable completion for zcash-cli(1)
-# Copyright (c) 2012-2016 The Bitcoin Core developers
+# bash programmable completion for bitcoinz-cli(1)
+# Copyright (c) 2017-2018 The BitcoinZ developers
```

Comment:
We must leave the original Copyrights

Reply:
The original copyrights are below, Github is just being dumb.